admin/app/components/organization-information-section.js

```diff
@@ -54,6 +55,12 @@ export default class OrganizationInformationSection extends Component {
     this.form = Form.create(getOwner(this).ownerInjection());
   }
 
+  get externalURL() {
+    const baseUrl = ENV.APP.BI_ORGANIZATION_DETAIL_DASHBOARD_URL;
+    const orgaUrl = baseUrl + this.args.organization.id;
```

Comment:
Si tu préfères avoir une variable je te propose organizationUrl sinon faire un return direct

Reply:
La version one-liner est un peu longue
`return ENV.APP.ORGANIZATION_DASHBOARD_URL && (ENV.APP.ORGANIZATION_DASHBOARD_URL + this.args.organization.id);`

J'ai nommé la variable baseUrl pour mettre en évidence que c'est une partie fixe (statique) et qu'on lui rajoute une donnée du contexte. OrganisationURL n'apporte pas cette information. Si je renomme en `urlDashboardPrefix`, c'est plus explicite ?

Reply:
Je pense que c'est plus explicite

Reply:
Corrigé